src/main/java/com/mongodb/jdbc/MongoDriver.java

```diff
@@ -279,8 +280,12 @@ private MongoConnection createConnection(ConnectionString cs, Properties info)
                             + ". Expected format <name>+<version>.");
         }
 
+        boolean extJsonMode =
+                Boolean.parseBoolean(info.getProperty(EXT_JSON_MODE.getPropertyName()));
```

Comment:
Nice, you were able to get the property set for `extjsonmode`.  

A boolean for the `value` may be a big confusing as both RELAXED and EXTENDED are technically still resulting in extended json.  Let's have the property take the values `RELAXED` or `EXTENDED` and obtain it as a string.  A similar example is with the `loglevel` or `logdir` values.  
ie
```
// Earlier in the file declare the RELAXED string, or an enum for RELAXED and EXTENDED
static final String RELAXED = "RELAXED";

// Get the value here, RELAXED as default
String extJsonModeVal = info.getProperty(EXT_JSON_MODE.getPropertyName(), RELAXED);

// set `extJsonMode` to true/false based on the string value otherwise throw exception if invalid value entered
```

Reply:
The `testClientInfoProperty()` example was helpful in writing a test :)